src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/index.tsx

```diff
@@ -131,36 +154,36 @@ const SettingsWidget = ({
           {!isLibrary && (
             <div className="my-3">
               <ResetCard
-                showResetButton={settings.showResetButton}
+                showResetButton={settings.showResetButton!}
```

Comment:
```suggestion
                showResetButton={settings.showResetButton || false}
```
Nit: this would be nicer than using `!` I think.